### PR TITLE
feat: cursors from clickhouse

### PR DESCRIPTION
**Pull request by blind-oracle**

### CODE REVIEW

File: k8s/cursor-initializator-clickhouse/cursor-initializator-clickhouse.py [hunk 1]
```diff
@@ -33,104 +22,31 @@ def get_logger():
...
-            checkpointer = os.path.join(path, "checkpoint.txt")
-            with open(checkpointer, "w", encoding="utf-8") as f:
-                f.write(aggregated[node][job]["cursor"] + "\n")
-                created += 1
+    for r in result.result_rows:
+        with open(os.path.join(args.output_dir, f"{r[0]}.txt"), "w", encoding="utf-8") as f:
+            f.write(f"{r[1]}\n")
 
-    logger.info("Successfully initialized cursors %s on path %s", created, args.output_dir)
+    logger.info(f"Successfully initialized {len(result.result_rows)} cursors on path {args.output_dir}")
```
Comment (sasa-tomic) [inline k8s/cursor-initializator-clickhouse/cursor-initializator-clickhouse.py:49]: Lazy log formatting is the recommended way to go: https://stackoverflow.com/questions/4148790/lazy-logger-message-string-evaluation

**Review by sasa-tomic — APPROVED** [review]
LGTM, but would be good to also get a review from @nikola-milosa

File: k8s/cursor-initializator-clickhouse/cursor-initializator-clickhouse.py (hunk 1 above)
Comment (blind-oracle) [reply k8s/cursor-initializator-clickhouse/cursor-initializator-clickhouse.py]: Ok, not that performance matters here :)

File: k8s/cursor-initializator-clickhouse/cursor-initializator-clickhouse.py [hunk 2]
```diff
@@ -33,104 +22,31 @@ def get_logger():
...
-            checkpointer = os.path.join(path, "checkpoint.txt")
-            with open(checkpointer, "w", encoding="utf-8") as f:
-                f.write(aggregated[node][job]["cursor"] + "\n")
-                created += 1
+    for r in result.result_rows:
+        with open(os.path.join(args.output_dir, f"{r[0]}.txt"), "w", encoding="utf-8") as f:
+            f.write(f"{r[1]}\n")
 
-    logger.info("Successfully initialized cursors %s on path %s", created, args.output_dir)
+    logger.info(f"Successfully initialized %d cursors on path %s", len(result.result_rows), args.output_dir)
```
Comment (sasa-tomic) [inline k8s/cursor-initializator-clickhouse/cursor-initializator-clickhouse.py:49]:
```suggestion
    logger.info("Successfully initialized %d cursors on path %s", len(result.result_rows), args.output_dir)
```

**Review by NikolaMilosa — APPROVED** [review]
Thank you @blind-oracle LGTM!

**Pull request merged**